### PR TITLE
implemented dynamic weather based on current location (IP)

### PR DESCRIPTION
**Pull request by kay-ou**
added dynamic weather based on current location (IP)
added sunrise and sunset time of current location
added hint for user to press spacebar for searching
replaced font by ubuntu

### CODE REVIEW

File: bookmarks.js [hunk 1]
```diff
@@ -13,35 +13,15 @@ const bookmarks = [
...
-        url: "https://reddit.com/r/pcmasterrace",
-      },
-      { name: "/r/me_irl", url: "https://reddit.com/r/me_irl" },
-      {
-        name: "/r/battlestations",
-        url: "https://reddit.com/r/battlestations",
-      },
-      { name: "/r/unixporn", url: "https://reddit.com/r/unixporn" },
-      { name: "/r/news", url: "https://reddit.com/r/news" },
+      { name: "TikTok", url: "https://tiktok" },
```
Comment (Jaredk3nt) [inline bookmarks.js:16]: Please don't make changes to the urls like here and in the README. These are fine in your fork but not in the primary repository

File: index.html [hunk 2]
```diff
@@ -56,34 +67,75 @@
...
+            } else {
+              console.log("error msg: " + xhr.status);
+            }
+          }
+        };
+        xhr.send();
+      }
+
+      // Display random jokes
+      function getRandomJokes(){
```
Comment (Jaredk3nt) [inline index.html:93]: Please don't add completely new functionality like this to the main repo without opening an issue first asking if it's okay. I don't think jokes fit with this projects intent

File: styles.css [hunk 3]
```diff
@@ -1,4 +1,5 @@
-@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+/* @import url('https://fonts.googleapis.com/css?family=Roboto+Mono'); */
+@import url('https://fonts.googleapis.com/css?family=Ubuntu:regular,bold&subset=Latin');
```
Comment (Jaredk3nt) [inline styles.css:2]: Please don't modify the design of the original project without first opening an issue and asking about the changes first. The project was designed to use Roboto.

**Review by Jaredk3nt — CHANGES_REQUESTED** [review]
Most of these changes seem like things you wanted in your own fork of the project and do not belong in the original. I'm glad to see people forking & experimenting with this but it's always best to open an issue first before just opening a PR with these kind of changes.

Also always attach screenshots of visual changes. I would rather not have to pull down your branch to check things out.

**Comment by kay-ou** [comment]
please forgive the newbie here 🤣 I didn't meant to merge so much. At the beginning I just wanted to add the dynamic weather api. but then I forgot to switch to my private repo, in which I changed a lot like adding dynamic public holidays api etc.. you could take a look here if you want: https://kayou.tk .I will close this request. You know what I mean, I'd be more than happy to help you get some new idea.

**Pull request closed without merging**